CHANGELOG.md

```diff
@@ -104,6 +105,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Extra shapes are drawn after Esc, or G pressed while drawing a region in grouping (<https://github.com/openvinotoolkit/cvat/pull/2507>)
 - Reset state (reviews, issues) after logout or changing a job (<https://github.com/openvinotoolkit/cvat/pull/2525>)
 - TypeError: Cannot read property 'id' of undefined when updating a task (<https://github.com/openvinotoolkit/cvat/pull/2544>)
+- Cannot find Python when deploying models through nuclio if steps are strictly followed in Installation_Automatic_Annotation.md
```

Comment:
Need to remove one line. I can see the same statement several times in CHANGELOG.md

Reply:
I just edited it in github web-client. Did it remove the duplicate?